.github/workflows/ci.yaml

```
@@ -25,7 +25,11 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --all-features -- -D warnings
+          args: --features external -- -D warnings
```

This should just build the default feature also right? As CI can't build with the `external` feature as it doesn't have an externally provided MoltenVK?

```suggestion
          args: -- -D warnings
```

---

Ooh oops your are right